docs/src/knownissues.md

```diff
@@ -36,9 +36,32 @@ This can be worked around be either:
 
 ## Open MPI
 
-### Segmentation fault
+### Segmentation fault when loading the library
 
-When attempting to use a system-provided Open MPI implementation, you may encounter a segmentation fault. This can be fixed by setting the environment variable `ZES_ENABLE_SYSMAN=1`. See [Open MPI issue #10142](https://github.com/open-mpi/ompi/issues/10142).
+When attempting to use a system-provided Open MPI implementation, you may encounter a segmentation fault upon loading the library, or whenever the value of an environment variable is requested.
+This can be fixed by setting the environment variable `ZES_ENABLE_SYSMAN=1`.
+See [Open MPI issue #10142](https://github.com/open-mpi/ompi/issues/10142) for more details.
```

Comment:
Is this still a problem in practice?  We try to work around it at https://github.com/JuliaParallel/MPI.jl/blob/096574032bfe6bde8ced4282f0ca458515c17dd2/lib/MPIPreferences/src/MPIPreferences.jl#L149 Is this not enough?

Reply:
That's only used when detecting the library. I think it may still be required when actually using MPI.jl